dyepack.cabal

```diff
@@ -16,9 +16,19 @@ maintainer:          matthewtpickering@gmail.com
 extra-source-files:  CHANGELOG.md
 
 library
-  -- exposed-modules:
+  exposed-modules:     Debug.Dyepack
   -- other-modules:
   -- other-extensions:
   build-depends:       base ^>=4.12.0.0
+                     , generics-sop
   hs-source-dirs:      src
   default-language:    Haskell2010
+  ghc-options:         -fwhole-archive-hs-libs -debug -O0
+
+executable example
+  main-is:             Main.hs
+  build-depends:       base ^>=4.12.0.0
+                     , dyepack
+  default-language:    Haskell2010
+  hs-source-dirs:      example
+  ghc-options:         -fwhole-archive-hs-libs -debug -O0
```

Comment:
I'll experiment where these flags are actually needed

Reply:
We don't need these in the library. Someone who uses the library needs them in their executable.